### DISCUSSION TITLE
Why Prometheus tries to scrape CephMGR's metrics on a wrong port

### DISCUSSION THREAD

**Discussion by ahmedTouati** [rook/rook#8303] [Q&A]
In my rook installation i can notice that we have these annotations on rook-ceph-mgr :

```
prometheus.io/path: /metrics
prometheus.io/port: "9283"
prometheus.io/scrape: "true"
```
which means that Prometheus will scrape metrics from port 9283, I can see that from Prometheus targets but the strange thing is that Prometheus tries to scrape ceph-mgr metrics on port 80 also which i haven't define

**Comment by travisn** [comment]
@ahmedTouati You see the metrics being collected as expected, right? So where do you see prometheus attempting to scrape from port 80?

**Reply by ahmedTouati** [reply]
@travisn I see metrics collected as expected from port 9283, but there is an additional endpoint on port 80 that prometheus tries to collect ceph-mgr metrics from as you can see in the screenshot :
![Capture](https://user-images.githubusercontent.com/50967678/126807784-a6e5abf1-7741-4907-92d0-42bee03460e6.PNG)

**Reply by travisn** [reply]
Not sure, @galexrt Any ideas?
